lib/database_connector.rb

```diff
@@ -26,8 +26,14 @@ def initialize
     initialize_admin_account
   end
 
-  private def initialize_tables
-    self.conn.exec(
+  def self.connection
+    @@connection
+  end
+
+  private
```

Comment:
although this is a valid syntax, but try to explicitly mention a private method in defintion just like in java, 
reason -> in future, if you add more methods, then it becomes hard to find out which is private and which is public